frontend/src/Components/loginForm.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import './login.css';
+
+export const LoginForm = () => {
+    const [username, setUsername] = useState<string>('');
+    const [password, setPassword] = useState<string>('');
+    
+    const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        try {
+          const response = await axios.post("http://localhost:8000/login/password", {
+            username,
+            password,
+          });
+          console.log(response.data)
+        } catch(error) {
+            console.error(error);
+        }
+    };
+    
+  const handleUsernameChange = (event: React.ChangeEvent<HTMLInputElement>, setUsername: React.Dispatch<React.SetStateAction<string>>) => {
+    setUsername(event.target.value);
+  };
+
+  const handlePasswordChange = (event: React.ChangeEvent<HTMLInputElement>, setPassword: React.Dispatch<React.SetStateAction<string>>) => {
+    setPassword(event.target.value);
+  };
+
+    return (
+        <section>
+          <img src='./logo192.png' alt='placeholder for wewrap logo'></img>
```

Comment:
depending on the timing of when this image will get provided, it's worth putting a comment here and then creating an entry in the task tracker to make sure we don't forget to complete this! it's very common in industry to have a comment that reads like:

// TODO (justin) T99: refactor this code after ____

Reply:
Done!